esp/src/src-react/components/Files.tsx

```diff
@@ -223,9 +243,11 @@ export const Files: React.FunctionComponent<FilesProps> = ({
             MaxSkew: {
                 label: nlsHPCC.MaxSkew, width: 60, formatter: (value, row) => value ? `${Utility.formatDecimal(value / 100)}%` : ""
             },
-            Modified: { label: nlsHPCC.ModifiedUTCGMT },
             Accessed: { label: nlsHPCC.LastAccessed },
```

Comment:
I'd think you'd want to have that same timezone toggle on this one (LastAccessed) also